### ISSUE TITLE
Kitcoin should be translated into other languages

### ISSUE THREAD

**Issue by edcous** [codeths/kitcoin#92]
**Describe what's wrong**
Kitcoin is currently in English only. We should have options for other languages to ensure accessibility.
**Describe what you'd like to change**
The extension especially and the website should have some sort of language change option.

**Comment by spaghetus** [comment]
I've transferred this issue to the Kitcoin issue tracker.

**Comment by edcous** [comment]
We are not currently planning to localize every string in Kitcoin. Given that automatic translators are on all major browsers, I am closing this issue.
